### PR TITLE
Adding line_number to Row

### PR DESCRIPTION
**Pull request by paulodeleo**
Adding line_number to Row so in an error message it is possible to show the line causing the error, this way:

`puts "Error on line #{import.report.invalid_rows.first.line_number}"`

This is PR #59 with the requested changes.

### CODE REVIEW

File: lib/csv_importer.rb [hunk 1]
```diff
@@ -69,8 +69,8 @@ def header
 
   # Initialize and return the `Row`s for the current CSV file
   def rows
-    csv.rows.map do |row_array|
-      Row.new(header: header, row_array: row_array, model_klass: config.model,
+    csv.rows.map.with_index do |row_array, index|
```
Comment (pcreux) [inline lib/csv_importer.rb]: `with_index(1)` will get the index start at 1. So you can do:

```
csv.rows.map.with_index(1) do |row_array, line_number|
  Row.new(..., line_number: line_number)
...
```

**Comment by pcreux** [comment]
This is great!

Can you add / update a test to ensure that the line number is correct?

File: lib/csv_importer.rb (hunk 1 above)
Comment (paulodeleo) [inline lib/csv_importer.rb]: Actually, it should be `.with_index(2)`, as the header line is not included in the rows array. The `report.invalid_rows` will never refer to an error on line 1 of the *file*. Agreed?

Comment (paulodeleo) [inline lib/csv_importer.rb]: Actually, it should be `.with_index(2)`, as the header line is not included in the rows array. The `report.invalid_rows` will never refer to an error on line 1 of the *file*. Agreed?

**Comment by paulodeleo** [comment]
Clising in favor of #63 .

**Pull request closed without merging**